src/main/java/io/gravitee/repository/jdbc/management/JdbcParameterRepository.java

```diff
@@ -123,8 +132,8 @@ public void delete(String key, String referenceId, ParameterReferenceType refere
     public Optional<Parameter> findById(String key, String referenceId, ParameterReferenceType referenceType) throws TechnicalException {
         LOGGER.debug("JdbcParameterRepository.findById({}, {}, {})", key, referenceId, referenceType);
         try {
-            final List<Parameter> items = jdbcTemplate.query("select * from parameters where " + escapeReservedWord("key") + " = ? and reference_type = ? and reference_id = ?"
-                    , ORM.getRowMapper()
+            final List<Parameter> items = jdbcTemplate.query(getOrm().getSelectAllSql() + " where " + escapeReservedWord("key") + " = ? and reference_type = ? and reference_id = ?"

```

Comment:
getSelectAll() escapes the table name.
```suggestion
            final List<Parameter> items = jdbcTemplate.query("select * from " + this.tableName + " where " + escapeReservedWord("key") + " = ? and reference_type = ? and reference_id = ?"

```